crates/meilisearch/Cargo.toml

```
@@ -50,6 +50,7 @@ jsonwebtoken = "9.3.1"
 lazy_static = "1.5.0"
 meilisearch-auth = { path = "../meilisearch-auth" }
 meilisearch-types = { path = "../meilisearch-types" }
+memmap2 = "0.9.7"
```

```suggestion
memmap2 = "0.9.5"
```

The rest of the code is using this version, I don't think we need to have both

---

The Crago.lock has been updated, I'll update the rest of the toml to switch to the latest version.